packages/Button/stories/examples/Basic.js

```diff
@@ -76,6 +78,9 @@ const ExampleStory = () => (
       <Button kind="link" onClick={clickHandler}>
         link
       </Button>
+      <Button.Link onClick={clickHandler} kind="primary" url="/">
```

Comment:
After you fix up the API, have a separate "section" for the Button.Link component for one that opens normally, and another that opens in a new tab.